Package.swift

```diff
@@ -37,8 +37,6 @@ let package = Package(
     products: [
         .library(name: "AzureCore", targets: ["AzureCore"]),
```

Comment:
this needs to be dynamic as well ?

Reply:
Sanath is testing without making this dynamic.

Reply:
It is removed.